Add info about julia meetup (#19)
Ferrite-FEM/FerriteCon: 2024/index.md

hasmath = false
+++

# FerriteCon 2024 & Julia Meetup Gothenburg

> This year we will combine FerriteCon 2024 with the first [Julia meetup](#julia_meetup_gothenburg) in Gothenburg, Sweden! Find the [program](#program) and information about [time and location](#time_and_location) below.  

Reviewer: ```suggestion
This year FerriteCon 2024 will be co-located with the first [Julia meetup](#julia_meetup_gothenburg) in Gothenburg, Sweden! Find the [program](#program) and information about [time and location](#time_and_location) below.
```

collaborations among Ferrite.jl users and developers, as well as to discuss
how the package can be further developed in the future.

[Chris Rackauckas](https://chrisrackauckas.com/) will give a keynote lecture about TBD.  

> The [program](#program) of the conference will consist of beginner-friendly presentations on Ferrite.jl and of current research
> using Ferrite.jl.

Reviewer: ```suggestion
The [program](#program) of the conference will consist of both beginner-friendly and advanced presentations on Ferrite.jl and of current research
using Ferrite.jl.
```

If you have another topic you would like to discuss, please feel free to do so,
as long as you can relate it to Ferrite.jl!

## Julia Meetup Gothenburg

> We are happy to announce the first Julia user meetup in Gothenburg, Sweden. The meetup will be held in conjunction with the FerriteCon, and will take place at [Chalmers University of Technology](https://www.chalmers.se/en). During the meetup, Julia famous [Chris Rackauckas](https://chrisrackauckas.com/) will give a keynote lecture on TBD.  

Reviewer: ```suggestion
We are happy to announce the first Julia user meetup in Gothenburg, Sweden. The meetup will be held in conjunction with FerriteCon, and will take place at [Chalmers University of Technology](https://www.chalmers.se/en). During the meetup [Dr. Chris Rackauckas](https://chrisrackauckas.com/) will give a keynote lecture (title TBD).  
```
